packages/x-telemetry/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "@mui/x-telemetry",
+  "version": "7.12.0",
+  "description": "MUI X Telemetry",
+  "author": "MUI Team",
+  "main": "src/index.ts",
+  "license": "SEE LICENSE IN LICENSE",
+  "bugs": {
+    "url": "https://github.com/mui/mui-x/issues"
+  },
+  "homepage": "https://mui.com/x/introduction/licensing/",
+  "publishConfig": {
+    "access": "public",
+    "directory": "build"
+  },
+  "scripts": {
+    "typescript": "tsc -p tsconfig.json",
+    "build": "pnpm build:modern && pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files && chmod +x build/scripts/cli.js",
+    "build:modern": "node ../../scripts/build.mjs modern",
+    "build:node": "node ../../scripts/build.mjs node",
+    "build:stable": "node ../../scripts/build.mjs stable",
+    "build:copy-files": "node ../../scripts/copyFiles.mjs && node ./scripts/addPackageScripts.js",
+    "build:types": "node ../../scripts/buildTypes.mjs",
+    "prebuild": "rimraf build tsconfig.build.tsbuildinfo"
+  },
+  "bin": {
```

Seeing :

```jsx
import { muiXTelemetrySettings } from '@mui/x-telemertry';

muiXTelemetrySettings.enableTelemetry(); 
```

made me think of #9092. https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free. Maybe we need:

```suggestion
  "sideEffects": 
  "bin": {
```

or

```suggestion
  "sideEffects": false,
  "bin": {
```